src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_variables.scss

```diff
@@ -57,6 +57,16 @@ $moving-column-menu-height: 24px / $font-size-normal * 1em;
 $moving-column-border-width: 5px;
 $annotation-width: 34%;
 
+//Buttons
+$button-height: 24px / $font-size-normal;
+$color-button-text: $color-text-highlight-introvert;
+$color-button-text-hover: $color-text-highlight-normal;
+$color-button-cta-text: $color-text-inverted;
+$color-button-cta-base: $color-text-highlight-normal;
+$color-button-cta-hover-background: $color-brand-two-extrovert;
+$color-button-cta-active-background: $color-brand-two-introvert;
```

Comment:
These variables are defined before the theme had a chance to overwrite the default variables. So if for example the theme overwrites `$color-test-highlight-normal`, `$color-button-text-hover` will still have the value from core.

To fix this, the variables need to be moved to a file that is loaded between `_variables_theme.scss` and `_button.scss`. Not sure how to do this without making it extremely complex.